pkgs/development/compilers/bambu/default.nix

```diff
@@ -0,0 +1,257 @@
+     fi
+    done
+
+  '';
+
+  # GCC used for libs and bintools for clang frontends
+  i686gcc = if gccFrontends != [ ] then builtins.head gccFrontends else pkgsi686Linux.gcc8;
+
+  # Combines clang and llvm binaries into the same directory
+  # This is necessary because bambu finds the LLVM tools by searching in the llvm directory
+  combineClangAndLLVM = llvmPackages: buildEnv {
+    name = "clang-with-llvm-" + llvmPackages.release_version;
+    paths = [
+      llvmPackages.clang
+      llvmPackages.clang.cc.dev
+      llvmPackages.clang.cc.libllvm
+      llvmPackages.clang.cc.libllvm.dev
+    ];
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    postBuild = ''
+      ln -s ${llvmPackages.clang}/bin/clang++ $out/bin/clang-cpp
+
+      for f in $out/bin/*; do
+        if [ -f "$f" ]; then
+          wrapProgram "$f" \
+            --set AR_FOR_TARGET ar \
+            --set AS_FOR_TARGET as \
+            --set CC_FOR_TARGET gcc \
+            --set CXX_FOR_TARGET g++ \
+            --set LD_FOR_TARGET ld \
+            --set NIX_BINTOOLS_FOR_TARGET ${i686gcc.bintools} \
+            --set NIX_BINTOOLS_WRAPPER_TARGET_TARGET_i686_unknown_linux_gnu 1 \
+            --set NIX_CC_FOR_TARGET ${i686gcc} \
+            --set NIX_CC_WRAPPER_TARGET_TARGET_i686_unknown_linux_gnu 1 \
+            --set NM_FOR_TARGET nm \
+            --set OBJCOPY_FOR_TARGET objcopy \
+            --set OBJDUMP_FOR_TARGET objdump \
+            --set RANLIB_FOR_TARGET ranlib \
+            --set READELF_FOR_TARGET readelf \
+            --set SIZE_FOR_TARGET size \
+            --set STRINGS_FOR_TARGET strings \
+            --set STRIP_FOR_TARGET strip \
+            --prefix HOST_PATH ":" ${i686gcc.bintools}/bin \
+            --prefix HOST_PATH ":" ${i686gcc}/bin \
+            --prefix PATH ":" ${i686gcc.bintools}/bin \
+            --prefix PATH ":" ${i686gcc}/bin \
+            --prefix NIX_LDFLAGS_FOR_TARGET " " "-L${llvmPackages.clang}/lib -L${llvmPackages.clang.cc.libllvm}/lib -L${pkgsi686Linux.buddy}/lib -L${libmpc}/lib -L${gmp}/lib -L${mpfr}/lib -L${boost}/lib -L${libxml2}/lib -L${xz}/lib -L${zlib}/lib -L${icu}/lib -L${suitesparse}/lib -L${mpfi}/lib -L${glpk}/lib  -L${i686gcc.cc}/lib" \
+            --prefix NIX_CFLAGS_COMPILE_FOR_TARGET " " "-I${llvmPackages.clang.cc.dev}/include -I${llvmPackages.clang.cc.libllvm.dev}/include -I${pkgsi686Linux.buddy}/include -I${libmpc}/include -I${gmp.dev}/include -I${mpfr.dev}/include -I${boost.dev}/include -I${libxml2.dev}/include -I${xz.dev}/include -I${zlib.dev}/include -I${icu.dev}/include -I${suitesparse.dev}/include -I${mpfi}/include -I${glpk}/include -I${i686gcc.cc}/include"
```

Comment:
Do them like this instead.

`lib.concatStringsSep " " (map (x: "-L${lib.getLib x}/lib") [ xz zlib ])`

`lib.concatStringsSep " " (map (x: "-I${lib.getDev x}/include") [ xz zlib ])`